ledger-tool/src/main.rs

```diff
@@ -2471,54 +2477,50 @@ fn main() {
             };
             let genesis_config = open_genesis_config_by(&ledger_path, arg_matches);
             let include_sysvars = arg_matches.is_present("include_sysvars");
-            let exclude_account_data = arg_matches.is_present("exclude_account_data");
             let blockstore = open_blockstore(
                 &ledger_path,
                 AccessType::TryPrimaryThenSecondary,
                 wal_recovery_mode,
             );
-            match load_bank_forks(
+            let (bank_forks, ..) = load_bank_forks(
                 arg_matches,
                 &genesis_config,
                 &blockstore,
                 process_options,
                 snapshot_archive_path,
-            ) {
-                Ok((bank_forks, ..)) => {
-                    let slot = bank_forks.working_bank().slot();
-                    let bank = bank_forks.get(slot).unwrap_or_else(|| {
-                        eprintln!("Error: Slot {} is not available", slot);
-                        exit(1);
-                    });
+            )
+            .unwrap_or_else(|err| {
+                eprintln!("Failed to load ledger: {:?}", err);
+                exit(1);
```

Comment:
It seems like this logic is redundant. Maybe it's not? Or maybe it wasn't always? But I've simplified it to just `bank_forks.working_bank()`.

Reply:
Ah, this doesn't line up as intended in the "Conversation" tab, but it does in the "Files changed" tab.